### ISSUE TITLE
Visual Studio 2019 support

### ISSUE THREAD

**Issue by reduckted** [mayerwin/vs-disable-no-source-tab#1]
This extension works in Visual Studio 2019 Preview 1.1, but it produces a warning, and it will not work in future Visual Studio 2019 updates.

When you open a solution, Visual Studio 2019 shows a warning:

> Visual Studio has detected one or more extensions that are at risk of not functioning in a future VS update.  [Learn more](https://blogs.msdn.microsoft.com/visualstudio/2018/05/16/improving-the-responsiveness-of-critical-scenarios-by-updating-auto-load-behavior-for-extensions/)

Following the instructions in that link and enabling logging, shows that this extension is the culprit.

> Extension 'Disable No Source Available Tab' uses the 'synchronous auto-load' feature of Visual Studio. This feature will no longer be supported in a future Visual Studio 2019 update, at which point this extension will not work. Please contact the extension vendor to get an update. (UI Context: {d2567162-f94f-4091-8798-a096e61b8b50} Package Guid: {5126c493-138a-46d7-a04d-ad772f6be155})

Switching to asynchronous auto-load is pretty straight-forward (though it's only supported from VS 2015 onwards). I've recently made these changes for some of my extensions, so I'm more than happy to submit a PR if you're still supporting this extension. Of course, if you don't want to support this extension any more, that's fine. 😄

**Comment by mayerwin** [comment]
Hi @reduckted, thanks for these details. I'd really appreciate a PR if you've already made the necessary changes for some of your other extensions and it is straightforward! 
I looked into it a while ago and it was rather confusing (I also wasn't sure it was worth spending time on it until the RC). So with your help for this extension, I'll learn how to implement the same changes with the two other extensions I have published. I am happy to keep supporting this extension (which I still use as of today). Too bad VS versions earlier than 2015 are not supported, but I'll publish a link to the latest vsix file supporting these older versions for users who cannot upgrade (as I did previously when incompatible changes were required). I'll look forward to your PR!
